nailgun/entities.py

```diff
@@ -7298,22 +6911,26 @@ def __init__(self, server_config=None, **kwargs):
             'api_path': 'katello/api/v2/status',
             'server_modes': ('sat'),
         }
-        super(Status, self).__init__(server_config, **kwargs)
+        super().__init__(server_config, **kwargs)
 
 
 class Subnet(
-        Entity,
-        EntityCreateMixin,
-        EntityDeleteMixin,
-        EntityReadMixin,
-        EntitySearchMixin,
-        EntityUpdateMixin):
+    Entity,
+    EntityCreateMixin,
+    EntityDeleteMixin,
+    EntityReadMixin,
+    EntitySearchMixin,
+    EntityUpdateMixin,
+):
     """A representation of a Subnet entity."""
 
     def __init__(self, server_config=None, **kwargs):
         self._fields = {
             'boot_mode': entity_fields.StringField(
-                choices=('Static', 'DHCP',),
+                choices=(
+                    'Static',
+                    'DHCP',
+                ),
```

Comment:
The earlier one was much better with some change as:
```suggestion
                choices=( 'Static',  'DHCP'),
```

Reply:
This is all formatted by Black, can't pick which of these we like an don't like.